cmd/pluginator/internal/builtinplugin/builtinplugin.go

```diff
@@ -93,7 +93,7 @@ func ConvertToBuiltInPlugin() (retErr error) {
 	}
 	w.write("")
 	w.write("func New" + root + "Plugin() resmap." + pType.String() + "Plugin {")
-	w.write("  return &" + root + "Plugin{}")
+	w.write("\treturn &" + root + "Plugin{}")
```

Comment:
this change is unrelated to the merging issue, but without it, `make generate-kustomize-builtin-plugins` changes the indentation in the generated plugin